site/app/templates/grading/simple/Display.twig

```diff
@@ -292,15 +292,16 @@
             <td class="option-small-input">
                 <input
                     class="option-small-box cell-grade" tabindex="0"
-                    style="text-align: center; {{ component_score == 0 ? "color: var(--standard-light-medium-gray);" : "" }}
+                    style="text-align: center; width: 120px; {{ component_score == 0 ? "color: var(--standard-light-medium-gray);" : "" }}
                         {{ component_score == 0 ? "background-color: var(--alert-background-blue);" : "background-color: var(--default-white);" }}"
-                    type="text"
+                    type="number"
```

Comment:
Setting `type="number"` would only allow to insert integer values in to the input field. 
I think user should also be able to enter floats as well.( I assume floats also be accepted based on [this ](https://github.com/Submitty/Submitty/blob/main/site/public/js/simple-grading.js#L436)). You could also use the attribute `min=0 ` to ensure that negative values are not inserted.